package.json

```diff
@@ -67,11 +67,11 @@
     "husky": "^4.0.3",
     "jest": "^24.8.0",
     "jest-nunit-reporter": "^1.3.1",
-    "nock": "^10.0.6",
+    "nock": "^12.0.3",
     "node-sass": "^4.13.1",
     "sass-lint": "^1.13.1",
     "supertest": "^4.0.2",
-    "testcafe": "^1.5.0",
+    "testcafe": "^15.1.317922",
```

Comment:
😂 don't know why this made me LOL

Reply:
it was through npm audit 🤷‍♀

Reply:
it is now broken though so need to revert